site/static/examples/tas/sample-queues.yaml

```
@@ -14,7 +14,7 @@ metadata:
   name: "tas-flavor"
 spec:
   nodeLabels:
-    cloud.provider.com/: "tas-node-group"
+    cloud.provider.com/topology-block: "tas-node-group"
```

```suggestion
    cloud.provider.com/node-group: "tas-node-group"
```
To better capture the intention as described in https://github.com/kubernetes-sigs/kueue/issues/3481#issuecomment-2464248854. 

cc @mwysokin